searchcore/src/vespa/searchcore/proton/server/lid_space_compaction_job_take2.h

```diff
@@ -0,0 +1,48 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "lid_space_compaction_job_base.h"
+#include <vespa/document/bucket/bucketspace.h>
+
+namespace storage::spi { class BucketExecutor;}
+namespace proton {
+    class IDiskMemUsageNotifier;
+    class IClusterStateChangedNotifier;
+}
+
+namespace proton::lidspace {
+
+/**
+ * Job that regularly checks whether lid space compaction should be performed
```

Comment:
Please update comment to indicate what is special about this implementation.

Reply:
Ok, will follow up in separate PR

Reply:
Fixed